x-pack/plugins/cases/server/authorization/utils.test.ts

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { nodeBuilder } from '../../../../../src/plugins/data/common';
+import { OWNER_FIELD } from '../../common';
+import {
+  combineFilterWithAuthorizationFilter,
+  ensureFieldIsSafeForQuery,
+  getOwnersFilter,
+  includeFieldsRequiredForAuthentication,
+} from './utils';
+
+describe('utils', () => {
+  describe('combineFilterWithAuthorizationFilter', () => {
+    it('returns undefined if neither a filter or authorizationFilter are passed', () => {
+      expect(combineFilterWithAuthorizationFilter()).toBeUndefined();
+    });
+
+    it('returns a single KueryNode when only a filter is passed in', () => {
+      const node = nodeBuilder.is('a', 'hello');
+      expect(combineFilterWithAuthorizationFilter(node)).toMatchInlineSnapshot(`
+        Object {
+          "arguments": Array [
+            Object {
+              "type": "literal",
+              "value": "a",
+            },
+            Object {
+              "type": "literal",
+              "value": "hello",
+            },
+            Object {
```

Comment:
I am curious, do you know what tis extra object is?

Reply:
Not sure actually 🤔  seems like it comes from here: https://github.com/elastic/kibana/blob/master/src/plugins/data/common/es_query/kuery/node_types/node_builder.ts#L16

Reply:
Perfect! I asked if case we where doing something wrong.